suave/offchain/blockstore.go

```diff
@@ -0,0 +1,32 @@
+package offchain
+
+import (
+	"context"
+	"io"
+
+	"github.com/ipfs/boxo/path"
+	"github.com/ipfs/go-cid"
+	iface "github.com/ipfs/kubo/core/coreiface"
+	"github.com/ipfs/kubo/core/coreiface/options"
+	"github.com/multiformats/go-multihash"
+)
+
+type Blockstore struct {
```

Comment:
why is this called blockstore? particularly asking because in a blockchain client it makes it sound like its for storing blocks

Reply:
agree, good point

Reply:
Yeah, it's a bad choice 😄  I'll rename it to something like `Blobstore`.